packages/vscode-extension/src/core/sdksetup/SdkProperties.ts

```diff
@@ -5,12 +5,44 @@
 'use strict';
 
 import { homedir } from 'os';
-import { join } from 'path';
-import { sdkDownloadUrl, sdkFilename } from './extension.config.json';
-
-export const SDK_DOWNLOAD_URL = sdkDownloadUrl;
-export const SDK_FILENAME = sdkFilename;
+import * as path from 'path';
+import * as fs from 'fs';

```

Comment:
Could we only add required methods from the libraries?

Reply:
Added required methods from the libraries.